README.md

```diff
@@ -1,34 +1,42 @@
 ## IES Syntax Highlighter
 
-The IES Syntax Highlighter is a Visual Studio Code (VS Code) extension designed to make working with IES in [Terse RDF Triple Language (TTL)](https://www.w3.org/TR/turtle/) (.ttl) files easier. This extension was created initially for internal use within the Data Science team at Telicent to make working with IES in TTL files nicer.
+A [Visual Studio Code](https://code.visualstudio.com) (VSCode) extension to help working with [IES](https://github.com/dstl/IES4) in [Terse RDF Triple Language (TTL)](https://www.w3.org/TR/turtle/) (.ttl) files
+
+Features:
+<ul>
+  <li>
+    <strong>Color highlighting</strong> - Highlights IES classes according to their defined colours<br>
+    <img src="./images/Overview.png" alt="colour highlighting screenshot" style="height:300px">
```

Comment:
Color (US) or colour (GB/Aus)... 🤔
Also on line 45.

Reply:
Good question.

Looks like its UK, unless the terminology comes from a standard written in US spelling

Andy G says: 

> UK spelling, for preference.
>
> There are some oddities around Catalog vs Catalogue, e.g. in the DCAT 3 standard.  I try and use their spellings when referring directly to it and UK spellings when talking about it.